.github/workflows/ci.yaml

```diff
@@ -224,12 +224,12 @@ jobs:
                 include:
                     - os: windows-latest
                       config: local
-                      bazelversion: 6.2.0
+                      bazelversion: 6.3.2
```

Comment:
6.2.0 seems to only be used for windows and osx. Update to 6.3.2, which also allows overriding repos in an e2e that the root overrides (bazel-lib in this case, for testing).

Reply:
Wow you've learned a lot during this fire 😂